Makefile

```diff
@@ -3,11 +3,21 @@ export TF_LOG=DEBUG
 SRC=$(shell find . -name '*.go')
 SONARQUBE_IMAGE?=sonarqube:latest
 SONARQUBE_START_SLEEP?=60
+GO_VER ?= go
 
 .PHONY: all vet build test
 
 all: fmt vet build
 
+tools:
+	cd tools && $(GO_VER) install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
+
+docs:
```

Comment:
are these rm's required? I'm guessing we have them here because `tfplugindocs` errors out when a file already exists?